vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -618,6 +633,16 @@ func (d *VolumeDriver) Unmount(r volume.UnmountRequest) volume.Response {
 		return volume.Response{Err: ""}
 	}
 
+	if plugin_utils.IsFullVolumeName(r.Name) != true {
```

Comment:
This check is unnecessary. fullVolumeNameMap[r.ID] if exists will have fullVolName